docs/src/configuration/app/app-reference.md

```diff
@@ -260,7 +260,7 @@ The following table presents possible properties for each location:
 | `index`             | Array of `string`s or `null`                         |           | Files to consider when serving a request for a directory. When set, requires access to the files through the `allow` or `rules` keys. |
 | `expires`           | `string`                                             | `-1`      | How long static assets are cached. The default means no caching. Setting it to a value enables the `Cache-Control` and `Expires` headers. Times can be suffixed with `ms` = milliseconds, `s` = seconds, `m` = minutes, `h` = hours, `d` = days, `w` = weeks, `M` = months/30d, or `y` = years/365d. |
 | `allow`             | `boolean`                                            | `true`    | Whether to allow serving files which don't match a rule. |
-| `scripts`           | `boolean`                                            |           | Whether to allow loading scripts in that location. Meaningful only on PHP containers. |
+| `scripts`           | `boolean`                                            |           | Whether to allow loading scripts in that location. Meaningful only on PHP containers. Only blocks scripts that are not in the `passthru`. |
```

Comment:
```suggestion
| `scripts`           | `boolean`                                            |           | Whether to allow scripts to run. Doesn't apply to paths specified in `passthru`. Meaningful only on PHP containers. |
```

Maybe not totally necessary, but I could only understand it by looking at it from a different perspective. Maybe it's the "blocks" + "not"?

Other possibility: Is overridden by paths specified in `passthru`.